INSTALL.md

```diff
@@ -0,0 +1,10 @@
+# Install Zync
+
+Download this repository and run `bundle install`.
+
+Install PostgreSQL with `brew install postgresql`
```

Comment:
this is clearly on for macOS right? 

maybe put it under a section ?

Reply:
I'd rather have a Brewfile than manual steps to install.

Reply:
@mikz Is it worth having a Brewfile for one single line? 🤔

Reply:
imho, yes. 

by convention, I know that a Brewfile should include _all_ dependencies. If I only see one, I know that is the _only_ dependency.